src/front-end/typescript/lib/framework/component/base.ts

```diff
@@ -62,6 +84,16 @@ export interface UpdateChildParams<
   mapChildMsg(msg: ChildMsg): ParentMsg;
 }
 
+/**
+ * Typically used in the context of an `update`
+ * function. Ensures the passing of the updated state
+ * from the parent to the child.
+ *
+ * @param params - the ParentState, ParentMsg, ChildState, and Child Message
+ * @returns The State and a Msg Array
+ * {@link UpdateChildParams}
+ * {@link UpdateReturnValue}
+ */
```

Comment:
Recommendation:

```
This function is used to update a child component's state inside
a parent component's update function.
```

and:

```
@returns UpdateReturnValue<PS, PM> //NOTE: not sure how to link docs here
```

Reply:
thank you...revised in https://github.com/bcgov/digital_marketplace/pull/332/commits/730fc6e16be84e1472a5ec3879fcea37596f920f